doc/nrf/external_comp/coremark.rst

```diff
@@ -0,0 +1,88 @@
+.. _integration_template:
+
+CoreMark integration
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+CoreMark is a CPU benchmark provided by EEMBC (Embedded Microprocessor Benchmark Consortium) to verify SoCs performance.
+It provides a series of test done in each benchmark iteration.
+The number of iterations DUT can perform per second gives benchmark score.
+Integrating CoreMark into |NCS| allows users to easily verify performance of acquired SoCs.
+
+
+Integration prerequisites
+*************************
+
+Before you start the |NCS| integration with CoreMark, make sure that the following prerequisites are completed:
+
+* :ref:`Installation of the nRF Connect SDK <installation>`.
+* Setup of the required :term:`Development Kit (DK)`.
+
+Solution architecture
+*********************
+
+CoreMark repository is added by default as a module to |NCS| by adding CoreMark repository to |NCS| west manifest.
```

Comment:
```suggestion
By adding the CoreMarek repository to the |NCS| west manifest, you can include it in the repository as the |NCS| module.
```

Reply:
User doesn't have to do anything he already has coremark repository thanks to west manifest.

Reply:
Edited the above suggestion - please double-check